core/templates/.spacemacs.template

```diff
@@ -252,8 +252,20 @@ values."
    ;; scrolling overrides the default behavior of Emacs which recenters point
    ;; when it reaches the top or bottom of the screen. (default t)
    dotspacemacs-smooth-scrolling t
-   ;; If non nil line numbers are turned on in all `prog-mode' and `text-mode'
-   ;; derivatives. If set to `relative', also turns on relative line numbers.
+   ;; Control line numbers activation.
+   ;; If set to `t' or `relative' line numbers are turned on in all `prog-mode' and
+   ;; `text-mode' derivatives. If set to `relative', line numbers are relative.
+   ;; This variable can also be set to a property list for finer control:
+   ;; '(:relative nil
+   ;; 	:disabled-for (dired-mode
+   ;; 	               doc-view-mode
+   ;; 	               markdown-mode
+   ;; 	               org-mode
+   ;; 	               pdf-view-mode
+   ;; 	               text-mode)
```

Comment:
You can use `spacemacs/mplist-get` instead of  `plist-get` like for the layer properties and then the list is not needed, the user car write:
 ```
:disabled-for dired-mode doc-view-mode ...
```